.github/workflows/focus-sync-strings.yml

```diff
@@ -2,45 +2,50 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/
 
-name: "Focus - Sync Strings"
+name: Focus - Uplift Strings
 
 on:
   schedule:
-    - cron: '0 2 * * *'
+    - cron: "0 2 * * *"
   workflow_dispatch:
 
 jobs:
   main:
-    name: "Sync Strings"
-    runs-on: ubuntu-20.04
+    name: Uplift strings
+    runs-on: ubuntu-latest
     steps:
-      - name: "Discover Focus Beta Version"
-        id: focus-android-beta-version
+      - name: Discover Focus beta version
+        id: discover-beta-version
         uses: mozilla-mobile/fenix-beta-version@4.2.2
-      - name: "Skip non-beta versions"
+      - name: Skip non-beta versions
         uses: andymckay/cancel-action@0.2
-        if: ${{ steps.focus-android-beta-version.outputs.beta_version == '' }}
-      - name: "Checkout Main Branch"
-        uses: actions/checkout@v2
+        if: ${{ steps.discover-beta-version.outputs.beta_version == '' }}
+      - name: Store ENV variables
+        id: env-variables
+        run: |
+          echo "project_name=Focus Strings" >> $GITHUB_OUTPUT
+          echo "beta_branch=releases_v${{ steps.discover-beta-version.outputs.beta_version }}" >> $GITHUB_OUTPUT
+          echo "branch_name=automation/uplift-focus-strings-${{ steps.discover-beta-version.outputs.beta_version }}" >> $GITHUB_OUTPUT
+      - name: Checkout default branch
+        uses: actions/checkout@v3
         with:
-          path: main
-          ref: main
-      - name: "Checkout Beta Branch"
-        uses: actions/checkout@v2
+          path: default
```

Comment:
Could we just stick with doing what this was doing before with the exact path and ref? I think that we have more clarity when we are explicitly saying we are checking out the main branch compare to default branch because that begs the question of what is a  default branch and how does the actions/checkout parameters work. 

At the end of the day, I will defer to your judgement since I think the l10n team should ideally just own these actions. 

We will have to start moving away from GitHub actions to utilize taskcluster around March as preparation for the m-c migration so the amount of effort we should spend here should be fairly minimal. My concern here is that we are losing clarity about what we are doing here with this change.